homeassistant/components/sensor/uber.py

```diff
@@ -97,16 +104,16 @@ def unit_of_measurement(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
-        price_details = self._product['price_details']
-        distance_key = 'Trip distance (in {}s)'.format(price_details[
-            'distance_unit'])
-        distance_val = self._product.get('distance')
-        if (price_details.get('distance_unit') is None) or \
-           (self._product.get('distance') is None):
-            distance_key = 'Trip distance'
-            distance_val = 'N/A'
-        time_estimate = self._product['time_estimate_seconds']
-        return {
+        price_details = self._product.get('price_details')
+        if price_details is None:
+            distance_key = 'Trip distance (in miles)'
+            distance_val = self._product.get('distance', 'N/A')
```

Comment:
It seems that setting the distance behavior is not influenced by the `if` statement.